packages/server/routes/api/events.js

```diff
@@ -114,4 +114,48 @@ router.get(
   }
 );
 
+router.get(
+  "/eventAttendees/:eventId",
```

Comment:
REST API naming says to use a hierarchy for the resources we provide in the API. This route will be on `/events`, and thinking from a design perspective, the attendees belong to the event with `:eventId`, so the route path here should be something like `/:eventId/attendees`.